packages/venia-ui/lib/components/MiniCart/footer.js

```diff
@@ -37,7 +36,7 @@ const Footer = props => {
                 subtotal={subtotal}
             />
             <Suspense fallback={placeholderButton}>
-                <Checkout cart={cart} setStep={setStep} step={step} />
+                <Checkout setStep={setStep} step={step} />
```

Comment:
This is because `Checkout` aka `Flow` gets `cart` from context now, correct?

Reply:
Yes.